co_sim_io/impl/vtk_utilities.hpp

```diff
@@ -0,0 +1,95 @@
+//     ______     _____ _           ________
+//    / ____/___ / ___/(_)___ ___  /  _/ __ |
+//   / /   / __ \\__ \/ / __ `__ \ / // / / /
+//  / /___/ /_/ /__/ / / / / / / // // /_/ /
+//  \____/\____/____/_/_/ /_/ /_/___/\____/
+//  Kratos CoSimulationApplication
+//
+//  License:         BSD License, see license.txt
+//
+//  Main authors:    Philipp Bucher (https://github.com/philbucher)
+//
+
+#ifndef CO_SIM_IO_VTK_UTILITIES_INCLUDED
+#define CO_SIM_IO_VTK_UTILITIES_INCLUDED
+
+// System includes
+#include <map>
+
+// Project includes
+#include "define.hpp"
+#include "macros.hpp"
+
+namespace CoSimIO {
+namespace Internals {
+
+// see https://vtk.org/wp-content/uploads/2015/04/file-formats.pdf ; figure 2
+enum class VtkCellType {
+    Vertex = 1,
+    Line = 3,
+    Triangle = 5,
+    Quad = 9,
+    Tetra = 10,
+    Hexahedron = 12,
+    Wedge = 13,
+    Pyramid = 14,
+
+    Quadratic_Edge = 21,
+    Quadratic_Triangle = 22,
+    Quadratic_Quad = 23,
+    Quadratic_Tetra = 24,
+    Quadratic_Hexahedron = 25,
+};
+
+// TODO make constexpr, static etc?
+inline std::map<VtkCellType, ElementType> GetVtkCellTypeToElementTypeMap()
+{
+    return {
+        {VtkCellType::Quadratic_Hexahedron,     ElementType::Hexahedra3D20},
+        {VtkCellType::Hexahedron,               ElementType::Hexahedra3D8},
+        {VtkCellType::Wedge,                    ElementType::Prism3D6},
+        {VtkCellType::Quad,                     ElementType::Quadrilateral2D4},
+        {VtkCellType::Quadratic_Quad,           ElementType::Quadrilateral2D8},
```

Comment:
@pooyan-dadvand I am not sure if we need the 2D types
What do you think?

=> it is not compatible with VTK, not sure if this should be a blocker

Reply:
To elaborate a bit more:
VTK doesn't support some of the types we support, hence this would get lost in translation

However I think we can solve this by introducing another field for the CoSimIO element-type, where we can do whatever we want
This would also be extensible in the sense that if we want in the future to support element types that are not at all compatible with vtk we can easily add it (but won't be able to load them in paraview)

From the implementation side this can be achieved easily

Reply:
So I would add them. Please note that the 2D ones can comes handy in mapping from 2D to 3D extruded models. (Also enabling our usual checks for in-plane conditions, etc.)

I think that for vtk we may not load them as you said or even convert them to the 3D versions.

Reply:
ok now it supports any element type independent from the vtk-standard
If we add in the future types that are not compatible with vtk, CoSimIO will still work, but paraview will not be able to visualize it.